### PR TITLE
[MM-29859] Improve "ltctl loadtest status" output

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

PR adds a couple of small improvements to `ltctl loadtest status` command to address feedback from users of the load-test tool:

- Added Running Time
- Added more information about errors.

New output looks like this:

```
==================================================
load-test status:

State: running
Start time: Mon Nov  2 17:51:17 UTC 2020
Running time: 3m24s
Active users: 632
Number of errors: 3131
  - Timeouts: 3131 (100.00%)
  - HTTP 5xx: 0 (0.00%)
  - HTTP 4xx: 0 (0.00%)
==================================================
```

#### Ticket

https://mattermost.atlassian.net/browse/MM-29859

### CODE REVIEW

**Comment by streamer45** [comment]
Build fails cause of `go-bindata` updates. The fix is included in https://github.com/mattermost/mattermost-load-test-ng/pull/392/commits/70d166eba266d4e8bae14adea303fa2af56bf4e2.

File: cmd/ltctl/loadtest.go [hunk 1]
```diff
@@ -5,14 +5,61 @@ package main
...
+	output, err := t.Output()
+	if err != nil {
+		return info, err
+	}
+
+	prometheusURL := fmt.Sprintf("http://%s:9090", output.MetricsServer.PublicIP)
+	helper, err := prometheus.NewHelper(prometheusURL)
+	if err != nil {
+		return info, fmt.Errorf("failed to create prometheus.Helper: %w", err)
+	}
```
Comment (agnivade) [inline cmd/ltctl/loadtest.go:48]: Maybe it's better to do this outside in the main function and just pass the helper object instead of `t`.

Comment (streamer45) [reply cmd/ltctl/loadtest.go:48]: I was trying to keep `RunLoadTestStatusCmdF` as clean as possible but that works too.

**Review by agnivade — APPROVED** [review]

**Review by isacikgoz — APPROVED** [review]
Nice one! 🎉

**Pull request merged**